<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Moles</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="MainMenu" name="mainMenu"/>
      <widget class="CreateMapMenu" name="createMapMenu"/>
      <widget class="CreateTeamMenu" name="createTeamMenu"/>
      <widget class="GameView" name="gameView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainMenu</class>
   <extends>QWidget</extends>
   <header location="global">mainmenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreateMapMenu</class>
   <extends>QWidget</extends>
   <header location="global">createmapmenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreateTeamMenu</class>
   <extends>QWidget</extends>
   <header location="global">createteammenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameView</class>
   <extends>QWidget</extends>
   <header location="global">gameview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
